numpy/lib/utils.py

```diff
@@ -1140,4 +1140,31 @@ def _opt_info():
             enabled_features += f" {feature}?"
 
     return enabled_features
+
+def drop_metadata(dt):
+    align = dt.isalignedstruct
+    if dt.names:
+        # structured dtype
+        l = list()
+        for n in dt.names:
+            t = dt.fields[n]
+            l.append((
+                n,  # name
+                drop_metadata(t[0]),  # format
+                t[1],  # offset
+                t[2] if len(t) == 3 else None,  # title
+            ))
```

Comment:
```suggestion
            if len(t) == 3:
                l.append((n, drop_metadata(t[0]), t[1], t[2]))

            else:
                l.append((n, drop_metadata(t[0]), t[1], None))
```
I think this is much more readable.